XenModel/InvisibleMessages.resx

```diff
@@ -138,6 +138,9 @@
   <data name="HOMEPAGE" xml:space="preserve">
     <value>http://www.citrix.com/xenserver</value>
   </data>
+  <data name="HOMEPAGE_FILENAME" xml:space="preserve">
+    <value>HomePage.mht</value>

```

Comment:
It would be good to add the equivalent line to the InvisibleMessages.(cn|jp).resx files, so the translators don't have to figure it out.